sdv/utils.py

```diff
@@ -0,0 +1,46 @@
+"""Utility functions."""
+from sdv._utils import _get_relationship_for_child, _get_rows_to_drop
+from sdv.errors import InvalidDataError
+
+
+def drop_unknown_references(metadata, data, drop_missing_values=True):
+    """Drop rows with unknown foreign keys.
+
+    Args:
+        metadata (MultiTableMetadata):
+            Metadata of the datasets.
+        data (dict):
+            Dictionary that maps each table name (string) to the data for that
+            table (pandas.DataFrame).
+        drop_missing_values (bool):
+            Boolean describing whether or not to also drop foreign keys with missing values
+            If True, drop rows with missing values in the foreign keys.
+            Defaults to True.
+
+    Returns:
+        dict:
+            Dictionary with the dataframes ensuring referential integrity.
+    """
+    metadata.validate()
+    try:
+        metadata.validate_data(data)
+        return data
```

Comment:
Can we also just add a unit test to make sure that valid data is returned as-is?